tests/test_csc.py

```diff
@@ -188,7 +188,7 @@ def test_applyCorrection(self):
             # Switch corrections off
             cmd_attr = getattr(harness.aos_remote, f"cmd_disableCorrection")
             send_topic = cmd_attr.DataType()
-            send_topic.all = True
+            send_topic.disableAll = True
```

Comment:
If you use ts_salobj 3.7 you can simplify this to:
```
harness.aos_remote.cmd_disableCorrection.set(all=True, disableAll=True)
await harness.aos.remote.cmd_disableCorrection.start(timeout=timeout)
```
and likewise all over the place. Unlike `RemoteTelemetry` and `RemoteEvent`, `RemoteCommand` has no `set_put` method (or `set_start`) because of the potential for conflicts with existing keyword arguments. But it's still one line instead of 3. And you can omit the set entirely for commands that have no arguments.